index.html

```diff
@@ -187,6 +187,39 @@ <h2>Skills</h2>
                     </ul>
                 </section>
             </section>
+            <section class="form-container">
+                <div class="form-title">
+                    <h3>I'm always interested in hearing about new projects, so if you'd like to chat please get in</h3>
+                </div>
+                <form action="https://formspree.io/f/xayajwko" method="POST">
+                    <ul>
+                        <li class="name">
+                            <label for="name"></label>
+                            <input type="text" id="name" name="user_name" placeholder="Full name" maxlength="30" required>
+                        </li>
+                        <li class="name">
+                            <label for="mail"></label>
+                            <input type="email" id="mail" name="user_email" placeholder="Email address" required>
+                        </li>
+                        <li class="textarea">
+                            <label for="msg"></label>
+                            <textarea id="msg" name="user_message" placeholder="Enter text here" maxlength="500" required></textarea>
+                        </li>
+                        <li class="button-container">
+                            <button type="submit">Get in touch</button>
```

Comment:
![image](https://user-images.githubusercontent.com/20563252/121273943-f9fdc080-c88e-11eb-9ce9-58f2d6ee3bd1.png)

The button should look enabled when the inputs are filled, disabled when one or more input are empty, and hover or pressed according to CSS style `:hover` or `:active`. I kindly invite you to add more CSS style on the button to follow this pattern on [figma portfolio](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471)